example.docker-compose.yml

```diff
@@ -0,0 +1,24 @@
+Version: '3'
+Service:
+  Pufferpanel:
```

Comment:
Should be using lowercase naming for all keys

Reply:
Preference, makes no functional difference.

Reply:
This is not a preference. Yaml is case sensitive. Docker might not care, but it is not correct, not does it match their docs. Use lowercase keys.

Reply:
keys made lowercase.